conf/Config.js

```diff
@@ -79,6 +79,16 @@ class Config extends EventEmitter {
         healthChecks.allowFrom =
             healthChecks.allowFrom.concat(defaultHealthChecks);
 
+        if (parsedConfig.redis &&
+          typeof parsedConfig.redis.sentinels === 'string') {
+            const redisConf = { sentinels: [], name: parsedConfig.redis.name };
+            parsedConfig.redis.sentinels.split(',').forEach(item => {
+                const [host, port] = item.split(':');
+                redisConf.sentinels.push({ host, port: parseInt(port, 10) });
```

Comment:
Please use `Number.pasrseInt` as the JS global is kinda looked down upon due it's magic behavior.